pkg/oc/cli/observe/observe.go

```diff
@@ -327,23 +329,58 @@ func (o *ObserveOptions) Complete(f kcmdutil.Factory, cmd *cobra.Command, args [
 		return err
 	}
 
-	switch o.templateType {
-	case "jsonpath":
-		p, err := NewJSONPathArgumentPrinter(o.includeNamespace, o.strictTemplates, o.templates...)
-		if err != nil {
-			return err
+	// TODO: Remove in the next release
+	// support backwards compatibility with misspelling of "go-template" output format
+	if o.PrintFlags.OutputFormat != nil && *o.PrintFlags.OutputFormat == "gotemplate" {
+		fmt.Fprintf(o.ErrOut, "DEPRECATED: The %q output format has been replaced by %q, please use the new spelling instead\n", "gotemplate", "go-template")
```

Comment:
sigh...

Reply:
@smarterclayton since this command is experimental are you ok with us fixing those without deprecation?